pkg/services/sqlstore/user.go

```diff
@@ -583,6 +584,55 @@ func (ss *SQLStore) GetSignedInUser(ctx context.Context, query *models.GetSigned
 	})
 }
 
+// GetTeamsByUser is used by the Guardian when checking a users' permissions
+func (ss *SQLStore) GetTeamsByUser(ctx context.Context, query *models.GetTeamsByUserQuery) error {
```

Comment:
Why are these methods in user service?

Reply:
sorry, user sqlstore

Reply:
Some other tests in sqlstore depend on these methods and I can't using team.Service because of a cycling import. Once user/org moves outside - they can import team.Service instead.

Reply:
Can you please write some comments, I lost overview what are the TODOs.